label_studio/projects/api.py

```diff
@@ -640,5 +642,34 @@ class ProjectModelVersions(generics.RetrieveAPIView):
     queryset = Project.objects.all()
 
     def get(self, request, *args, **kwargs):
+        # TODO make sure "extended" is the right word and is
+        # consistent with other APIs we've got
```

Comment:
I think this should be addressed before merging

Reply:
I think we don't have consistent naming convention regarding request args